<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_cargar_pdf_base">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargar PDF Base</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_seleccionar_archivos">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccionar Archivos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_elegir_Carpeta_destino">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ejegir carpeta destino</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_archivos_seleccionados">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cargar_pdf_base">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_carpeta_destino_elegida">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ejecutar_programa">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Ejecutar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_indicar_trabajo_terminado">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
